### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Nepomuceno**
Correcting commit for version 0.1 release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,14 +23,15 @@ if you want this to be unique for this module and not shared with other instance
...
+  source  = "Azure/naming/azurerm"
   suffix = [ "test" ]
 }
 resource "azurerm_resource_group" "example" {
   name     = module.naming.main.resource_group.name
   location = "West Europe"
 }
 ```
-There are other advances usages that will be explained in the [Advanced usages](#advancedusages) part og this docs.
+Other advances usages will be explained in the [Advanced usage](#advancedusage) part of this docs.
```
Comment (jjcollinge) [inline README.md:34]: advances -> advanced

File: README.md [hunk 2]
```diff
@@ -41,13 +43,47 @@ There are other advances usages that will be explained in the [Advanced usages](
 
 ## Modifying resources
 
-The resources are automatically generated using `go` to change the generation please change the file on the `templates` folder. To add a new resource, include its definition in the file `resourceDefinition.json`, and it will be automatically generated when `main.go` is run.
+The resources are automatically generated using `go` to change the generation please change the file on the `templates` folder. To add a new resource, including its definition in the file `resourceDefinition.json`, and it will be automatically generated when `main.go` is run.
```
Comment (jjcollinge) [inline README.md:46]: Might need rewording a little, hard to follow.

File: README.md [hunk 3]
```diff
@@ -41,13 +43,47 @@ There are other advances usages that will be explained in the [Advanced usages](
...
 
 You can find a list bellow of all the resources that are currently implemented. To get a list of the ones that are missing implementation you can check at [Missing resources](docs/missing_ressources.md) the resources that have no documentation about their limitation on naming currently on Microsoft docs are on the [Not defined](docs/not_defined.md) list.
 
 
+# Advanced usage
+
+
+## Output
+
+Each one of the resources spits out more than just the name of the resource but also other properties:
```
Comment (stevehaigh) [inline README.md:58]:
```suggestion
Each one of the resources emits the name of the resource and other properties:
```

File: README.md [hunk 4]
```diff
@@ -41,13 +43,47 @@ There are other advances usages that will be explained in the [Advanced usages](
...
+
+| Property | Type | Description |
+| ----- |----- | ---- |
+| name | string | name of the resource including their respective suffixes and prefixes applied |
```
Comment (stevehaigh) [inline README.md:62]:
```suggestion
| name | string | name of the resource including respective suffixes and prefixes applied |
```
